src/theory/bv/bvgauss.cpp

```diff
@@ -29,24 +29,34 @@ namespace CVC4 {
 namespace theory {
 namespace bv {
 
-static bool is_bv_const(Node n)
+namespace {
+
+bool is_bv_const(Node n)
 {
   if (n.isConst()) { return true; }
   return Rewriter::rewrite(n).getKind() == kind::CONST_BITVECTOR;
 }
 
-static Node get_bv_const(Node n)
+Node get_bv_const(Node n)
 {
   Assert(is_bv_const(n));
   return Rewriter::rewrite(n);
 }
 
-static Integer get_bv_const_value(Node n)
+Integer get_bv_const_value(Node n)
 {
   Assert(is_bv_const(n));
   return get_bv_const(n).getConst<BitVector>().getValue();
 }
 
+// Returns x - y if the result would be non-negative. Otherwise, returns 0.
+unsigned SubtractIfNonNegative(const unsigned x, const unsigned y)
```

Comment:
This function is overkill, in particular since it is only called once.
Replace its call with its return statement.

Reply:
Done.